README.md

```diff
@@ -94,7 +94,8 @@ some_ros_package
 |   | <b>MyNode.h</b>
 | 
 └───test
-|   | <b>my-node-test.cpp</b>
+|   | <b>my_node_test.cpp</b>
+|   | <b>sample_package_test.test</b>
```

Comment:
Isn't a rostest still testing the **node** rather then the **package**? perhaps a better name would be `my-node-rostest.test`? (again, the `-` rather then the `_` for naming convention is another issue)